src/java-runtime/index.ts

```diff
@@ -175,14 +181,18 @@ export async function findJavaRuntimeEntries(): Promise<JavaRuntimeEntry[]> {
   return Promise.all(_.map(entries, async entry => {
     if (!await checkJdkInstallation(entry.path)) {
       entry.isValid = false;
-      entry.hint = "This path does not point to a JDK.";
+      entry.hint = "This path is not pointing to a JDK.";
+      const pathObject = path.parse(entry.path||"");
+      if (pathObject.name && pathObject.name.toLowerCase() === "bin") {
```

Comment:
Case insensitive comparison is what I really wanted in the first place...

Reply:
any concern of using `path.basename(string)` instead of `path.parse(string)`?

Reply:
works too. But let's move on...